docs/api.rst

```diff
@@ -11,19 +11,48 @@ API
 Time
 ^^^^
 
+All time-related types use units defined by a :ref:`Clock`. The default :ref:`Scheduler` :ref:`Clock` uses milliseconds as its units: :ref:`Time`, :ref:`Delay <Delay-type>`, :ref:`Period`, and :ref:`Offset` will all be millisecond values.
+
```

Comment:
if milliseconds are defined here do we need to repeat them for each one individually ?

Reply:
There are probably better ways to structure things.  My thinking was that repeating it in multiple spots will be helpful.  I was considering two main ways people use documentation:

1. They read or skim parts of it like a book to understand concepts, or get a feel for the API, etc.
2. They look for the answer answer a specific question, like "what are the units of the number I pass to `delay`?"

Repetition is slightly annoying for number 1, but imho, very helpful for number 2.

For number 1, they'll likely encounter the `Time` heading first, and read that time units are milliseconds when using the default `Scheduler`.  My thinking is that they'll then just tend to ignore the repetition when they encounter it later.

For number 2, someone will tend just to go look for the `delay` function (possibly finding it by readthedocs search, or just find-on-page text search).  At that point they'll find a link to the `Delay` type, hopefully click on it, and see that its units are milliseconds.  If `Delay` doesn't say that it's milliseconds, the person would have to take another step: find the `Time` heading and read the intro prose.